docs/modules/tiered-storage/pages/compactor.adoc

```diff
@@ -0,0 +1,26 @@
+= Compactor
```

Comment:
There is a lot of info here that I'm not sure we need to detail. Instead, here's a suggested shortened section for the compactor:

The Tiered Storage compactor operates on each data partition to reclaim disk space by removing unused data.

When the configured threshold of available disk space is surpassed, compaction is initiated synchronously before and after a Map operation. To optimize the compaction process, the compactor scans the area with the highest garbage ratio to reclaim disk space efficiently. During this process, live entries discovered by the compactor are moved to memory, which may temporarily reduce the efficiency of the hot cache. 

WDYT @blazember, @Serdaro, @oliverhowell, @rebekah-lawrence - do we need much more?

We should keep the section on the fine-tuning.

Reply:
Less is more. Seems good to me.

Reply:
Addressed in https://github.com/hazelcast/hz-docs/pull/1047/commits/bb91f6376439c66396bc062a1d906f1a15b64397